addons/ophthalmology/XEH_postInit.sqf

```diff
@@ -0,0 +1,41 @@
+#include "script_component.hpp"
+
+if (!hasInterface) exitWith {};
+
+call FUNC(initKeybinds);
+
+
+["KAT_EyewasherItem", "KAT_Eyewasher"] call ACEFUNC(common,registerItemReplacement);
+
+["CBA_settingsInitialized", {
+    if (!GVAR(enable) && !GVAR(enableManualBlinking)) exitWith {};
```

Comment:
but wouldnt that mean that if someone enable this addon, but dont want manual blinking, there wont be dust injury effect?

Reply:
No it would not